Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:6
+MAINTAINER code.vire@gmail.com
+
+RUN npm install -g yarn
+
+COPY package.json /tmp
+RUN cd /tmp &&\
+    yarn &&\
+    mkdir /app &&\
+    mv node_modules /app/node_modules &&\
+    rm package.json
+
+WORKDIR /app
+COPY . /app
+
+ARG REACT_APP_BACKEND_URL # env variable to specify API URL durin build
```

Comment:
Thanks for the hint! I've added the comment to tell why I'm passing a variable `REACT_APP_BACKEND_URL` during build time...